utils/usbmuxd/Makefile

```diff
@@ -8,18 +8,21 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=usbmuxd
-PKG_VERSION:=1.1.1
-PKG_RELEASE:=2
+PKG_VERSION:=1.1.2
```

Comment:
I don’t think PKG_VERSION makes sense here.

Reply:
@neheb 
Thanks Rosen! Indeed having date and commit returned by `--version` makes more sense. I updated all 6 packages that are now using a git commit. I rebuilt and tested them. Here's how the version is reported now:
``` bash
plistutil 2.4.0
inetcat 2.1.0
ideviceinfo 2024.04.16~5f083426
idevice_id 2024.04.16~5f083426
usbmuxd # no version reported no mater if PKG_VERSION is set or not
irecovery 1.2.0
idevicerestore 2024.04.04~6d40d0ab
ifuse 2023.01.06~814a0e38
ideviceinstaller 2024.01.15~22872c35
ideviceactivation 2024.04.16~6925d58e
```